packages/hardhat-core/src/internal/hardhat-network/provider/node.ts

```diff
@@ -870,6 +887,7 @@ Hardhat Network's forking functionality only works with blocks from at least spu
       irregularStatesByBlockNumber: this._irregularStatesByBlockNumber,
       userProvidedNextBlockBaseFeePerGas:
         this.getUserProvidedNextBlockBaseFeePerGas(),
+      emptyBlockRanges: this._blockchain.getEmptyBlockRanges(),
```

Comment:
I'm not really sure if this is necessary. Wouldn't the latestBlock be enough?

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/2032/commits/8a96e90c9c37326b3ed35ecb1433da4a13458128

Reply:
Missed some cleanup in that last commit.  Done here: https://github.com/nomiclabs/hardhat/pull/2032/commits/8134fb818354c42bda395b1b2d90d96c2d1c6976